README.md

```diff
@@ -1 +1,23 @@
-# drizzle
\ No newline at end of file
+# Drizzle
+
+> A pattern library authoring environment
+
+## Install
+
+0. [Download](https://github.com/cloudfour/drizzle/archive/master.zip)
+0. `npm start`
+
+## Overview
+
+Drizzle is a project template and build tool for creating pattern libraries. It outputs static HTML that can live anywhere, using [Gulp], [PostCSS], [Babel], [Webpack] and [Handlebars] under the hood.
+
+## Inspiration
+
+- https://fbrctr.github.io
+- http://solid.buzzfeed.com
```

Comment:
I might also add [Pattern Lab](http://patternlab.io/), which influenced a lot of the changes we requested or made ourselves to Fabricator.